### PR TITLE
Return otpAuthUrl from set API.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -192,6 +193,14 @@ api.set = async ({
   }
   if(secret) {
     rValue.secret = secret;
+    // produces a URL like `otpauth://totp/...`
+    rValue.otpAuthUrl = authenticator.keyuri(
+      // user identifier
+      email || account,
```
Comment (dlongley) [inline lib/index.js]: Do you think we should always bind this to `account` instead? (which means looking it up before hand if we don't have the ID).

File: lib/index.js [hunk 2]
```diff
@@ -192,6 +193,14 @@ api.set = async ({
...
+      // a service name, `undefined` is OK if `clientId` was not passed in
+      clientId,
```
Comment (dlongley) [inline lib/index.js]: If we put `clientId` here, the totp token will only work with that specific client ... I'm not convinced we want that. Is that intentional? It is, of course, optional (if no `clientId` is passed) ... but is that a feature we want to support?

Comment (dlongley) [reply lib/index.js]: Sounds like this should be `appId` or `serviceId` instead (a new param).

File: lib/index.js (hunk 1 above)
Comment (mattcollier) [reply lib/index.js]: This is added here for the benefit of the authenticator APP that uses the URL, I don't know how it's used.  However, we can assume that the authenticator App may have knowledge of email addresses, but certainly would not know anything about the account ID (urn:uuid:...)

File: lib/index.js (hunk 2 above)
Comment (mattcollier) [reply lib/index.js]: Done.

**Pull request merged**